opencga-app/app/scripts/docker/opencga-init/setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# TODO
+# Change hbase.client.keyvalue.maxsize to 0 in /etc/hbase/conf/hbase-site.xml on hadoop cluster to avoid 'KeyValue size too large' error
+# by sshing to your (hdinsight cluster e.g. ssh sshuser@bartcga-ssh.azurehdinsight.net) and changing the xml (e.g. vi /etc/hbase/conf/hbase-site.xml)
```

Comment:
@bart-jansen is this something we need to do before we'll be able to use HBase correctly, I seem to remember it was but I might be wrong.

Reply:
@lawrencegripper Correct, at least before you index large loads using opencga. 

This one-liner should do the trick: 
```sudo sed -i '/<name>hbase.client.keyvalue.maxsize<\/name>/!b;n;c<value>0</value>' /etc/hbase/conf/hbase-site.xml```